bert_reranker/main.py

```diff
@@ -34,14 +39,17 @@ def main():
     parser.add_argument('--output', help='where to store models', required=True)
     parser.add_argument('--no-model-restoring', help='will not restore any previous model weights ('
                                                      'even if present)', action='store_true')
-    parser.add_argument('--validation-only', help='will not train - will just evaluate on dev',
+    parser.add_argument('--train', help='will not train - will just evaluate on dev',
+                        action='store_true')
+    parser.add_argument('--validation', help='will not train - will just evaluate on dev',
```

Comment:
I think `validate` is better than `validation` (I know I wrote that variable name :) )